apps/teams-test-app/src/components/VisualMediaAPIs.tsx

```diff
@@ -0,0 +1,104 @@
+import { visualMedia } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const imagesHelper = (medias: visualMedia.VisualMediaFile[]): string => {
+  let message = '';
+  for (let i = 0; i < medias.length; i++) {
```

Comment:
Using `for..of` is preferred because it is more idiomatically recognized in JavaScript and it works more broadly (e.g. over `Iterable`s, so it's a more uniform construct.

Reply:
Thanks a lot for both of your suggestion, good for me to learn!!

Reply:
I agree with @AE-MS we should use `for..of` for iterating.